apps/01PigLatin.js

```diff
@@ -9,10 +9,45 @@ var rl = readline.createInterface({
 
 
 function pigLatin(word) {
-
+  word = word.toLowerCase();
   // Your code here
+  var vowelIndex = -1; // Set it to assume there are no vowels
+
+  if ( ( word.indexOf('a') > -1 && word.indexOf('a') < vowelIndex ) || vowelIndex === -1 ) {
+      vowelIndex = word.indexOf('a');
+    }
+
+  if ( ( word.indexOf('e') > -1 && word.indexOf('e') < vowelIndex ) || vowelIndex === -1 ) {
+      vowelIndex = word.indexOf('e');
+    }
+
+  if ( ( word.indexOf('i') > -1 && word.indexOf('i') < vowelIndex ) || vowelIndex === -1 ) {
+      vowelIndex = word.indexOf('i');
+    }
+
+  if ( ( word.indexOf('o') > -1 && word.indexOf('o') < vowelIndex ) || vowelIndex === -1 ) {
+      vowelIndex = word.indexOf('o');
+    }
+    
+  if ( ( word.indexOf('u') > -1 && word.indexOf('u') < vowelIndex ) || vowelIndex === -1 ) {
+      vowelIndex = word.indexOf('u');
+    }
+
+  if ((vowelIndex === 0) || (vowelIndex === -1))
+    {
```

Comment:
Unlike the rest of your if statements, this '{' doesn't immediately follow the conditions.